pkgs/Nukesor/pueue/pueued/scaffold.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/aqua-generate-registry.json
+# aqua - Declarative CLI Version Manager
+# https://aquaproj.github.io/
+# Other than name is optional. All initial values are just examples.
+name: Nukesor/pueue/pueued
+version_filter: not (Version matches "-rc")
+# version_prefix: cli-
+all_assets_filter: Asset matches "^pueued-"
```

Comment:
```suggestion
all_assets_filter: not (Asset matches "^pueue-" or Asset == "systemd.pueued.service")
```

Please don't use such a filter as this excludes assets such as `checksums.txt`.
It's hard to check if assets such as `checksums.txt` aren't used.
You should exclude `pueue-*` and `systemd.pueued.service` explicitly.

https://github.com/Nukesor/pueue/releases/tag/v4.0.0

Reply:
I'm sorry.
Next time I'll be sure to take into consideration what you pointed out.
Thank you for your comment.